src/types.d.ts

```diff
@@ -159,6 +160,9 @@ export interface ScatterplotMethodOptions {
     transitionDuration: number;
     transitionEasing: (t: number) => number;
     preventFilterReset: boolean;
+    hover: number;
+    select: number | number[];
+    focus: number | number[];
```

Comment:
nit; The addition of the properties could benefit from more descriptive names such as hoveredPoints, selectedPoints, and focusedPoints to enhance clarity. I will let you decide here, as you know the codebase consistency better

Reply:
I will keep these prop names as is as they match the function names and internally the `draw()` function literally calls `hover()`, `select()`, or `focus()` instead of directly setting the state.